ipaserver/plugins/internal.py

```diff
@@ -953,6 +970,175 @@ class i18n_messages(Command):
             "truncated": _("Query returned more results than the configured size limit. Displaying the first ${counter} results."),
             "unselect_all": _("Unselect All"),
         },
+        "ssbrowser": {
+            "ssbrowser-msg": _(
+                "<h1>Browser Kerberos Setup</h1>\n"
+                "\n"
```

Comment:
Is it possible to use some markdown? I think there shouldn't be too much HTML-tags in translations.

Reply:
Also it is not good idea to use this text as gettext key.
So it can be for example _('SSBrowser-page-content').
And in .po file we can put this conent:
msgid "SSBrowser-page-content"
msgstr "Some raw text with tags..."

Reply:
No problem, as you wish.

Reply:
About markdown... it can be done later. But still there is needed to make compact gettext key (example is above) and provide translations in po-files.

Reply:
"But still there is needed to make compact gettext key (example is above) and provide translations in po-files."

I don't understand you.
The Gettext key is here already.
After pushing an updated pot file to zanata, translators will fix outdated translations.
Then ipa release person will merge to.

Isn't it?

Reply:
Please
> make **compact** gettext key

There are several options how to do that. I have proposed one above.
Other option, you can split ssbrowser-page-content translation into 4 pieces:
* title
* firefox section
* chrome section
* ie section

What you choose is up to you.

Reply:
So, as you proposed I split out long keys.

Reply:
Ok, thank you, looks good.